packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -83,9 +88,15 @@ class UserSessionManager implements IUserSessionManager {
       state.loadOptions.value.sessions.autoTrack === false || sessionInfo.manualTrack === true
     );
 
-    const sessionTimeout: number = isNumber(state.loadOptions.value.sessions.timeout)
-      ? (state.loadOptions.value.sessions.timeout as number)
-      : DEFAULT_SESSION_TIMEOUT;
+    let sessionTimeout: number;
+    if (!isNumber(state.loadOptions.value.sessions.timeout)) {
```

Comment:
Shouldn't timeout also be a positive integer?

Reply:
Addressed